app/src/main/java/org/fossasia/susi/ai/activities/SettingsActivity.java

```diff
@@ -68,27 +76,66 @@ public boolean onPreferenceClick(Preference preference) {
 
                 @Override
                 public boolean onPreferenceClick(Preference preference) {
-                    AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
+                    final AlertDialog.Builder builder = new AlertDialog.Builder(getActivity());
```

Comment:
@amitiwary999 Its better to use a list preference in the case of settings instead of AlertDialog.